Engine/src/Application.bf

```diff
@@ -0,0 +1,79 @@
+using SteelEngine.Window;
+using SteelEngine.Events;
+
+namespace SteelEngine
+{
+	/*
+	Usage:
+
+	class Sandbox : Application
+	{
+		public override void OnInit()
+		{
+			base.OnInit();
+		}
+	
+		public override void OnCleanup()
+		{
+			base.OnCleanup();
+		}	
+	}
+
+	static void Main()
+	{
+		var app = scope Sandbox();
+		app.Run();
+	}
+
+	*/
```

Comment:
Is this meant to show a use case for how this class would be inherited? If so, can we move it to an example project instead of having a big comment showing how to do it?